tests/rbd/rbd_utils.py

```diff
@@ -511,6 +520,21 @@ def image_unmap(self, device_name):
         """UnMap provided rbd image."""
         return self.exec_cmd(cmd=f"rbd unmap {device_name}", output=True)
 
+    def rbd_bench(self, **kw):
+        """
+        This method writes IOs using rbd bench
+        Args:
+            **kw: test data
+                io: Amount of IO to be written
+                imagespec: pool/image on which IO needs to be run
```

Comment:
Add optional comment on default io size is 500G

Reply:
Done.